doc/md/writing-motoko/modules-and-imports.md

```diff
@@ -1,23 +1,29 @@
+---
+sidebar_position: 16
+---
+
 # Modules and imports
```

Comment:
I wonder if we should actually be referring to base more consistently as a package (a collection of modules), not a library, or at least indicate that we use the terms library and package interchangeably.